docs/data-modeling.md

```diff
@@ -0,0 +1,66 @@
+# Model your data
+Data modeling (i.e., design your database schemas) is the process of conceptualizing and visualizing how data will be stored and used by identifying the patterns used to access data and the types of queries to be performed within business operations. 
+
+This page first explains the ScalarDB data model and then describes how to design your database schemas based on the data model.
+
+## ScalarDB data model
+ScalarDB’s data model is an extended key-value model inspired by the Bigtable data model. It is similar to the relational model but differs in several ways, as described below.
+
+The following diagram shows an example of ScalarDB tables, each a collection of records. This section first explains what objects, such as tables and records, ScalarDB defines and then describes how to locate records.
+
+![ScalarDB data model.](images/scalardb_data_model.png)
+
+### Objects in ScalarDB
+The ScalarDB data model has several objects.
+
+#### Namespace
+A namespace is a collection of tables analogous to an SQL namespace or database. Typically, each application creates all its tables in one namespace.
+
+#### Table
+A table is a collection of data. A namespace most often contains one or more tables, each identified by a name. Each table includes a set of partitions.
+
+#### Partition
+A partition is a collection of records and a unit of (logical) distribution to nodes. Therefore, records within the same partition are placed in the same logical location.
+
+#### Record / Row
+A record or row is a set of columns that is uniquely identifiable among all of the other records.
+
+#### Column
+A column is a fundamental data element and does not need to be broken down any further. Each record is composed of one or more columns. Each column has a data type. For details about the data type, refer to [Data-type mapping between ScalarDB and other databases](https://scalardb.scalar-labs.com/docs/latest/schema-loader#data-type-mapping-between-scalardb-and-other-databases).
+
+#### Index
+An index is a copy of the records in a single table, sorted by a column. ScalarDB queries use indexes to find data more efficiently in a table, given the values of a particular column. 
```

Comment:
I feel something wrong with the expression "a copy of the records" since it's not a whole copy of the record. It might be something like "a copy of a certain single column that has (logical?) links for the original records".

Reply:
Right, it's a description for clustered indexes.
ScalarDB indexes are unclustered indexes, so I will update the description.
Thanks!